### DISCUSSION TITLE
How do I add negative versions of translate?

### DISCUSSION THREAD

**Discussion by DanielBailey-web** [tailwindlabs/tailwindcss#5690] [Help] [answered]
I am trying to extend translate values to include the following:
```
      translate: {
        "1/5": "20%",
        "2/5": "40%",
        "3/5": "60%",
        "4/5": "80%",
        "1/7": "14.2857143%",
        "2/7": "28.5714286%",
        "3/7": "42.8571429%",
        "4/7": "57.1428571%",
        "5/7": "71.4285714%",
        "6/7": "85.7142857%",
      }
```
Which adds the positive values, but how do I get access to the negative ones? (e.g. -translate-y-2/5)

**Comment by dance2die** [comment, marked as answer]
You can add negative counter parts with `"-2/5": "-40%"`  
**Demo**: https://play.tailwindcss.com/4FjQvxbhEs?file=config

      translate: {
        '1/5': '20%',
        '2/5': '40%',
        '-2/5': '-40%',
        '3/5': '60%',
        '4/5': '80%',
        '1/7': '14.2857143%',
        '2/7': '28.5714286%',
        '3/7': '42.8571429%',
        '4/7': '57.1428571%',
        '5/7': '71.4285714%',
        '6/7': '85.7142857%',
      },

**Reply by DanielBailey-web** [reply]
Amazing, thanks for the help!
